util.sh

```diff
@@ -0,0 +1,839 @@
+#!/usr/bin/env bash
```

Comment:
This is quite a lot of boilerplate we're copying over.  Do we really need all this? Shouldn't kubetest be able to handle cluster bringup?

Reply:
I've asked about kubetest, and the answers weren't encouraging. Using it outside of the kubernetes/kubernetes repo is untested. It's also a legacy project which will get replaced with a kubetest2 rewrite. I'd rather not rely on it.

Regarding util.sh, that's actually not needed for cluster bringup. It's for verify-shellcheck.sh.

Reply:
How are we going to keep this up to date if kubernetes changes it?  Could we potentially vendor this in instead?

Reply:
One cannot vendor shell scripts, and even if we could, the version from Kubernetes assumes that it runs in k/k, so a verbatim copy wouldn't work for us. In practice I expect that we'll just keep using the current copy unmodified until something breaks and once that happens, we can see if it is something that has been fixed upstream and manually apply the fix.